content/participate/review-changes.adoc

```diff
@@ -8,21 +8,30 @@ tags:
   - review
 ---
 
-NOTE: This page is under development, there will be more content added soon.
-See the jira:WEBSITE-662[] EPIC for tasks related to this page, contributions are welcome!
-
 Help us to review changes to code or documentation!
+All pull requests within Jenkins GitHub organizations are publicly open for reviews, and any reviews will be much appreciated.
 
 == Code Reviews and Copy Editing
 
-You can help review changes to the code and the documentation through following lists:
+Contributors and component maintainers may explicitly request request reviews from other community members.
+Below there are some queries which can help to find such open pull requests where help is needed:
 
-- link:https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+team%3Ajenkinsci%2Fcode-reviewers[Open pull requests for code reviews]
-- link:https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+team%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing]
+- link:https://github.com/search?q=is%3Aopen+is%3Apr+team-review-requested%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing], mostly for Jenkins website
```

Comment:
When I ran the proposed query from a "Guest Window" in Chrome, it included many more organizations than 
```suggestion
- link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+is%3Aopen+is%3Apr+team-review-requested%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing], mostly for Jenkins website
```

Reply:
That's strange, but it will not hurt